test_scripts/WebEngine/WebEngineProjection/010_OnSCU_after_Show_widget.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@LitvinenkoIra There are the same comments as for previous test script.

Reply:
@aderiabin Done in b2655a1d4abd36817f0f99596d92e656817ac74d